packages/sanity/src/_internal/cli/commands/build/buildCommand.ts

```diff
@@ -5,7 +5,7 @@ const helpText = `
 Options
   --source-maps Enable source maps for built bundles (increases size of bundle)
   --no-minify Skip minifying built JavaScript (speeds up build, increases size of bundle)
-  -y, --yes Use unattended mode, accepting defaults and using only flags for choices
+  -y, --yes Unattended mode, use 'y' for prompts and use only flags for choices
```

Comment:
Personally feel it would be clearer with something like

```suggestion
  -y, --yes Unattended mode, answers "yes" to any "yes/no" prompt and otherwise uses defaults
```